src/storage-preview/azext_storage_preview/__init__.py

```diff
@@ -84,6 +88,18 @@ def register_path_argument(self, default_file_param=None, options_list=None):
         self.ignore('file_name')
         self.ignore('directory_name')
 
+    def register_path_argument_v2(self, default_file_param=None, options_list=None):
```

Comment:
could you just replace previous register_path_argument()?

Reply:
OK~